### DISCUSSION TITLE
Updated through docker, and now not loading

### DISCUSSION THREAD

**Discussion by sassanix** [docmost/docmost#392] [Help / Questions] [answered]
I did the latest update, and it stopped working. 

The homepage is completely blank. 

Please advice on what I should do.

**Comment by sassanix** [comment]
Here are my logs:

```
> docmost@0.4.1 start /app

> pnpm --filter ./apps/server run start:prod

> server@0.4.1 start:prod /app/apps/server

> cross-env NODE_ENV=production node dist/main

(node:41) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

[Nest] 41  - 10/11/2024, 11:54:59 PM     LOG [NestFactory] Starting Nest application...

[Nest] 41  - 10/11/2024, 11:54:59 PM     LOG [DatabaseModule] Establishing database connection

[Nest] 41  - 10/11/2024, 11:54:59 PM     LOG [DatabaseModule] Database connection successful

[Nest] 41  - 10/11/2024, 11:54:59 PM     LOG [DatabaseMigrationService] No pending database migrations

[Nest] 41  - 10/11/2024, 11:54:59 PM     LOG [NestApplication] Nest application successfully started
```

When I did the command:

```
/app $ node --trace-deprecation
Welcome to Node.js v21.7.3.
```

**Comment by Philipinho** [comment]
Are you using any reverse proxy?

**Reply by Philipinho** [reply]
Check your browser console for any errors.

**Reply by sassanix** [reply]
I am using cloudflare for the reverse proxy, I can visit the /login page, but when I login it just goes to the /home and it's blank.

The console only shows ws connect, nothing else.

**Comment by sassanix** [comment]
I see this error now:

415 Unsupported Media Type

![image](https://github.com/user-attachments/assets/4e377629-4f72-4b50-b835-0384428b416f)

**Reply by Philipinho** [reply]
Can you try accessing it without Cloudflare proxy?

**Reply by sassanix** [reply]
It works without cloudflare, awesome!

**Comment by sassanix** [comment, marked as answer]
I got the reverse proxy working as well.

I enabled these two settings.

![image](https://github.com/user-attachments/assets/255d4b22-ff93-49ef-b78e-eb96f1631c4f)
![image](https://github.com/user-attachments/assets/4ef6b0e5-1cff-4f0f-a7e4-036e9d5112fd)

**Reply by bram-nijenkamp** [reply]
I have disabled these as well.... using cloudflare tunnels and traefik behind it. But still don't have it working yet, any other suggestions?
<img width="1664" alt="image" src="https://github.com/user-attachments/assets/ba72a9e9-5ebe-47a1-af13-3300dca92d04">

**Reply by Philipinho** [reply]
@FrozenSource did you disable the options after taking this screenshot?

**Reply by bram-nijenkamp** [reply]
sorry off course meant I turned these settings on like the answer suggests.

**Reply by Philipinho** [reply]
Any errors on your browser console?

**Reply by bram-nijenkamp** [reply]
yeah the same 415 on the “me” request which afterwards off course throws an error.

**Reply by Node815** [reply]
I have the same on a fresh install (Today)  as well.  Same 415 error with the 'me' request, also under Cloudflare Tunnels.  It used to work, but stopped.  Not sure when, but about a month ago, too our instance offline due to a staffing change with our Online crew we used to have for a game and am replacing it for our personal use.   For this, I wiped the entire data directory and let it install it fresh.  Locally, I can access it all day long, but not from external web such as  docmost.mydomain.com. 

I tried the workarounds above except for not using it behind Cloudflare.

**Reply by acmg** [reply]
It worked for me only by activating "Disable Chunked Encoding".

**Reply by bram-nijenkamp** [reply]
Yeah it's fixed for me as well now! just the disable chunked encoding.

**Comment by silicoflare** [comment]
If you use an old cloudflared tunnel, and do not want to migrate in the dashboard, add this to the `config.yml` file:

```yaml
originRequest:
  disableChunkedEncoding: true
```
